calblueprint/dccentralkitchen
# Start-up, Sign-in and Login Styling

[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
Styled the 3 Auth screens according to the Figma
- `StartUpScreen` (new!)
   - Vertical alignment for `ButtonLabel` in the works
   - Icons are not added yet
- `LoginScreen`
- `SignUpScreen`



## Relevant Links

### Online sources

[//]: # "Optional - copy links to any tutorial or documentation that was useful to you when working on this PR"
- `TextInput` documentation: https://reactnative.dev/docs/textinput#onfocus
- React dynamic keys: https://medium.com/@bretdoucette/understanding-this-setstate-name-value-a5ef7b4ea2b4

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"
- NIL
## How to review

[//]: # "The order in which to review files and what to expect when testing locally"
- Check Figma to see if everything matches
- Check if styling is still valid with hardware of different sizes, and if anything should be made responsive
- Code quality and improvements (redundant code/imports etc.)
   - Suggestions for better variable/component naming are welcome!
## Next steps

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"
- Remove `LastName` field -- to be done in tandem with current logic when creating a new customer / Airtable columns
- Back button in `LoginScreen` and `SignUpScreen`
- Green verification ticks after user has entered information in each TextInput
- Icons at `StartUpScreen`
- Number formatting into aaa-bbb-cccc
- Vertical alignment of buttons

## Tests Performed, Edge Cases
[//]: # "Hopefully we will add a testing suite/CI soon, but until then note down the steps you took to test locally"
- Logging in as existing customer and signing up as new customer working.

### Screenshots

[//]: # "Add screenshots of expected behavior - GIFs if you're feeling fancy!"
- `StartUpScreen`
![image](https://user-images.githubusercontent.com/35573990/76043001-1c29c580-5f0b-11ea-9827-639478bf73ce.png)
- `SignUpScreen`
![image](https://user-images.githubusercontent.com/35573990/76043160-85a9d400-5f0b-11ea-8aa2-65bcba426dc0.png)

- LoginScreen
![image](https://user-images.githubusercontent.com/35573990/76043135-7460c780-5f0b-11ea-986d-a93190069bc6.png)

CC: @wangannie @thumn 

[//]: # "This tags in both Annies as a default. Feel free to change, or add on anyone who you should be in on the conversation."
